c2cgeoportal/scripts/c2ctool.py

```diff
@@ -419,8 +419,8 @@ def step3(self):
                     "This is an optional step but it helps to have a standard project.\n"
                     "In the `create.diff` file, there are all the changes made in the create template "
```

Comment:
While I am on it...:

The `create.diff` file is a recommendation of the changes that you should apply to your project